src/main/java/Duke.java

```diff
@@ -1,10 +1,41 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.io.File;
+import java.io.IOException;
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Constructor for Duke.
+     */
+    public Duke() {
+        String filePath = new File("").getAbsolutePath();
+        int ind = filePath.lastIndexOf("/");
+        assert ind == -1 : "File path is incorrect";
+        String path = filePath.substring(0, ind + 1);
+        filePath = path.concat("duke.txt");

```

Comment:
Consider creating a final variable that stores this info and pass that variable.